.github/workflows/pypi-release.yml

```diff
@@ -24,6 +29,4 @@ jobs:
         run: python setup.py sdist bdist_wheel
 
       - name: Publish distribution 📦 to PyPI
-        uses: pypa/gh-action-pypi-publish@release/v1
-        with:
-          password: ${{ secrets.pypi_password }}
+        uses: pypa/gh-action-pypi-publish@release/v1
```

Comment:
EOF missing

Reply:
Thanks for raising that. Hope that I have made the correct changes now by adding the upload endpoint for PyPi.

Reply:
EOF still missing in that file